src/pipeline_v3/cli/management.py

```
@@ -588,6 +609,103 @@ def _parse_index_type(self, index_type_str: str) -> IndexType:
         else:
             raise ValueError(f"Invalid index type: {index_type_str}")
 
+    def _migrate_deprecated_parameters(self, args):
```

Ruff (0.11.9)

Missing return type annotation for private function `_migrate_deprecated_parameters`

(ANN202)